homeassistant/components/sensor/solaredge.py

```diff
@@ -0,0 +1,176 @@
+"""
+Support for SolarEdge Monitoring API.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/sensor.solaredge/
+"""
+
+from datetime import timedelta
+import logging
+
+import voluptuous as vol
+
+from homeassistant.components.light import PLATFORM_SCHEMA
```

Comment:
Don't import `PLATFORM_SCHEMA` from the light component. Import it from the sensor component.